examples/lighting-app/silabs/SiWx917/src/AppTask.cpp

```diff
@@ -38,14 +38,23 @@
 
 #include <platform/CHIPDeviceLayer.h>
 
-#define APP_FUNCTION_BUTTON &sl_button_btn0
-#define APP_LIGHT_SWITCH &sl_button_btn1
+#ifdef ENABLE_WSTK_LEDS
+#include "LEDWidget.h"
+#endif // ENABLE_WSTK_LEDS
+
+#ifdef ENABLE_WSTK_LEDS
+#define APP_ACTION_LED 1
+#endif // ENABLE_WSTK_LEDS
```

Comment:
Regroup them under the same #ifdef there is nothing in between. It reduces vertical spacing and reduces the number of #ifdef to read

Reply:
sure